business/clustering.go

```diff
@@ -0,0 +1,160 @@
+package business
+
+import (
+	"github.com/kiali/kiali/config"
+	"github.com/kiali/kiali/kubernetes"
+)
+
+// ClusteringService is a support service for retrieving data about the mesh environment
+// when Istio is installed with multi-cluster enabled.
+type ClusteringService struct {
```

Comment:
I wonder if we maybe we should just call this MeshService as a way to query anything we find relevant to a Mesh.

Reply:
Yes, I think your suggested renaming is OK, as this may include some things not related to clustering. I'll do it.